### PR TITLE
A couple more updates from the PR Sara started

### PR DESCRIPTION
**Pull request by watkyn**

### CODE REVIEW

**Review by macblazer — COMMENTED** [review]
Maybe need some modification based on the recently merged PRs.

**Review by macblazer — APPROVED** [review]

File: .swiftlint.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+opt_in_rules:
+  - sorted_imports
+  - trailing_closure
+
+file_length:
+  ignore_comment_only_lines: true
+
+disabled_rules:
+  - trailing_whitespace
```
Comment (hisaac) [inline .swiftlint.yml:9]: Could we remove this? I think leaving this enabled to warn of trailing whitespace is a fine idea.

**Review by hisaac — APPROVED** [review]
I'm fine approving, but I do think we should remove `trailing_whitespace` from the `disabled_rules` section.

File: .swiftlint.yml (hunk 1 above)
Comment (watkyn) [reply .swiftlint.yml:9]: yeah, it's annoying how Xcode sometimes handles this, which is why I added it. Personally I like things tidy as well, so I'll fix up all these places too.

**Pull request merged**